app/views/like/_like.html.erb

```diff
@@ -103,6 +103,11 @@
     .like-buttons {
         margin-top:5px;
     }
+
```

Comment:
Hey @preeity, Thanks so much for opening this. A few changes though. I don't believe we need this change for this issue, could you kindly remove the changes made to this file? Thanks